sodetlib/operations/uxm_setup.py

```diff
@@ -363,6 +363,35 @@ def setup_tune(S, cfg, bands, show_plots=False, update_cfg=True):
     return True, summary
 
 
+def setup_fixed_tones(S, cfg, freq, tone_power=10, update_cfg=True):
```

Comment:
We probably also need an automated procedure to find the frequencies to use based on gaps where resonances don't exist from the tunefiles or find_freq results.